### DISCUSSION TITLE
Changes made to bracket/map pool aren't fully applied until client restart

### DISCUSSION THREAD

**Discussion by ILW8** [ppy/osu#19095] [Ideas] [answered]
2) In response to a request for feedback from running the lazer tournament client for [The Roundtable](https://osu.ppy.sh/wiki/en/Tournaments/The_Roundtable)
____
This happened during the 1v1 match between fieryrage and KoaLeahQ. I swapped the bracket teams' order right before the first map was played. The changes didn't show up in the gameplay and mappool windows, however once a winner was determined, the winner window reflected the updated bracket. 

- [Bracket being changed right before a map starts](https://jame.btmc.live/feed_tournamentclient_james_2022-07-09_17-50-53.ts.trimmed_startat.h264_cq27.trimmed.mp4)
- [Video of "wrong" winner being shown](https://jame.btmc.live/feed_tournamentclient_james_wrong_winner.mp4)

Don't think it's a big deal, though a more consistent behavior to how changes to a bracket are applied is welcome.

**Comment by peppy** [comment]
I'll look into this one, but can you confirm that the duplication of screen content in this area is from your capture setup, and not something lazer-side?

![Safari 2022-07-13 at 08 53 31](https://user-images.githubusercontent.com/191335/178693152-4b755177-525f-49ab-9a3e-9d4c10d67063.png)

**Reply by ILW8** [reply]
Confirmed caused by capture and not lazer-sided.

**Comment by peppy** [comment, marked as answer]
Resolved via #19101.
